train.py

```diff
@@ -162,7 +192,8 @@
 if config["enable_wandb"]:
     run = wandb.init(
         entity="nasa-harvest",
-        project="ai4food-challenge",
+        project=config['project'],
+        name=config['name'],
```

Comment:
Is this optional? Ie will it work if not specified?

Reply:
Yes, the new params I added (--name, --unique, --project) all work as how it did before if left unspecified.  In other words, it will assign a random name and send it off to your wandb storage, not mine.